packages/Webkul/Checkout/src/Cart.php

```diff
@@ -243,6 +244,10 @@ public function updateItems($data)
                 continue;
             }
 
+            if ($item->product && $item->product->status === 0) {
+                throw new \Exception(trans('shop::app.checkout.cart.item.inactive'));
```

Comment:
Please use either `__()` or `trans()`, to keep consistency. `__()` is preferred, though :)

Reply:
Done